### PR TITLE
Initial integration with the stack analysis service

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
Nah, typo :)

File: vars/mavenCanaryRelease.groovy [hunk 1]
```diff
@@ -12,6 +12,15 @@ def call(body) {
     sh "mvn org.codehaus.mojo:versions-maven-plugin:2.2:set -U -DnewVersion=${config.version}"
     sh "mvn clean -e -U deploy"
 
+    if (flow.hasService("bayesian")) {
+        try {
+          sh 'mvn io.github.stackinfo:stackinfo-maven-plugin:0.1:prepare'
+          def response = bayesianAnalysis()
+        } catch (err) {
+          echo 'Unable to run Bayesian analysis'
```
Comment (rawlingsj) [inline vars/mavenCanaryRelease.groovy]: maybe we should log the error too, something like 
```
echo "Unable to run Bayesian analysis: ${err}"
```

Comment (msrb) [inline vars/mavenCanaryRelease.groovy]: Yeah, that's a good idea. Updated 😉

**Comment by rawlingsj** [comment]
nice @msrb :)

**Pull request merged**